### DISCUSSION TITLE
How to have thumbnails for videos on an instance hosted using Yunohost?

### DISCUSSION THREAD

**Discussion by Terkyz** [LycheeOrg/Lychee#1939] [Q&A]
Hello, I'm just starting into the world of selfhosting, and honestly I don't know pretty much at this moment, but I want to learn.

So, I started using Yunohost, but I really don't have a single idea how to enable thumbnails for videos, on mp4 and webm. I already have FFmpeg installed on Yunohost, but that's it, I haven't modified anything.

Thanks in advance and sorry if this is not the place to ask.

**Comment by ildyria** [comment]
For that one, it might be better to directly ask the people on Yunohost.
https://yunohost.org/en/app_lychee
